15-draft.json

```diff
@@ -18,6 +18,15 @@
       "description": "The name of your space",
       "type": "string"
     },
+    "type": {
+      "description": "The type of your space. Clients MUST treat a missing <code>type</code> field as if it were set to <code>space</code>. <code>space</code>: A hacker- or makerspace. For endpoints with <code>\"type\": \"space\"</code>, the <code>location</code> is mandatory. <code>hub</code>: An association of multiple people or spaces, which does not have a physical location itself.  Examples of hubs could be umbrella organizations or groups without a single hacker- or makerspace.",
```

Comment:
What about virtual spaces, like groups of people that meet in varying locations (e.g. a LUG), but without a fixed room? I think those could fit into the SpaceAPI model.

Reply:
Agree, allowing "virtual" spaces (such without a fixed location) to take part in the spaceapi covers way more entities as the differentiation between space and hub. The later imO is independently from whether its a physical or a virtual space and should be an own (optional) attribute (like `is_hub` or such). The current proposal doesn't allow for spaces to also be hubs, which should be considered a valid case.

Reply:
We discussed that the type field can be removed if the location becomes optional.